runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java

```diff
@@ -464,13 +516,17 @@ private String locateGo() {
 		return prop;
 	}
 
-	private String locateRuntime() {
+	private static File locateRuntime() {
 		final ClassLoader loader = Thread.currentThread().getContextClassLoader();
 		final URL runtimeSrc = loader.getResource("Go");
```

Comment:
I was kidding. :) 

I wouldn't have known it either if it hadn't already been there...

```
wjk@empit:antlr4$ grep -R "final URL runtimeSrc = loader.getResource" *
runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java:		final URL runtimeSrc = loader.getResource("Go");
runtime-testsuite/test/org/antlr/v4/test/runtime/javascript/node/BaseTest.java:		final URL runtimeSrc = loader.getResource("JavaScript/src");
runtime-testsuite/test/org/antlr/v4/test/runtime/python/BasePythonTest.java:		final URL runtimeSrc = loader.getResource(targetName+"/src");
```

The constant is not used _instead_, the constant is resolved against "the resource", together forming the absolute path to the package directory.